### DISCUSSION TITLE
Support for Spell Alerts

### DISCUSSION THREAD

**Discussion by CoreVale** [SFX-WoW/Masque#10] [Questions] [answered]
I love masque!!

I want to find a way to change the scale (and?) texture of the cooldown spark and glow around proc'ed (activated) abilities on my actionbars. I'm using a masque skin which makes my icons smaller than default, this is great- however the cooldown spark and glow around proc'ed abilities is still set to default (100%) scale which looks horrendous. Also, I hear masque used to support font changes, any way to reinstate that?

not sure where to change these things, halp!

thanks

**Comment by StormFX** [comment, marked as answer]
Masque does support adding custom textures using:

`Masque:AddSpellAlert("Shape", "Glow", "Ants")`

Though you'd have to give it a unique name ("Shape") and create the textures ("Glow" is the flash texture and "Ants" is the animated spark) yourself. They can't be scaled without messing things up, so it'd be better to create the textures at the scale you want.

As far as fonts go, people have always used all-in-one font add-ons so Masque having font options seemed/seems a bit redundant.

**Comment by CoreVale** [comment]
ok!, thanks for the quick reply- so Ive got the textures worked out,  but I would love to be able to configure text (font,scale,tags like monochrome outline) independently from other addons through mask,  could u point me in the right direction?

**Comment by StormFX** [comment]
For fonts you'll have to set those manually. Sorry. Take a look at Tekticles or something.
